src/app/work-item/work-item-list/work-item-list-entry/work-item-list-entry.component.scss

```diff
@@ -5,9 +5,10 @@
   cursor: pointer;
   .workItemList_workItemType {
     span{
-      font-size: $font12;
+      font-size: $font14;
```

Comment:
@nimishamukherjee @sanbornsen This change is not related to this PR. Please remove it.